README.md

```diff
@@ -73,16 +70,19 @@ make install
 Run `mongocli help` for a list of available commands
 or check our [online documentation](https://docs.mongodb.com/mongocli/master/) for more details.
 
-### Getting API Keys
+### Configuring MongoCLI with Atlas
+To use `mongocli` with Atlas you'll need to run `mongocli auth login` and follow the prompted steps on your terminal.
+
+### Configuring MongoCLI with Ops Manager and Cloud Manager
 
-To use `mongocli` you'll need to get API keys, to get them follow the documentation 
-appropriate for the service you're using, 
+#### Getting API Keys (Ops manager / Cloud Manager)
+To use `mongocli` you'll need to get API keys, to get them follow the documentation
+appropriate for the service you're using,
 [Atlas](https://docs.atlas.mongodb.com/configure-api-access/),
 [Ops Manager](https://docs.opsmanager.mongodb.com/current/tutorial/configure-public-api-access/),
 or [Cloud Manager](https://docs.cloudmanager.mongodb.com/tutorial/manage-programmatic-api-keys/)
 
-### Configuring `mongocli`
-
+#### Set up your credentials
 Run `mongocli config` to set up your credentials, 
 this is optional, you can use [env variables](https://docs.mongodb.com/mongocli/stable/configure/environment-variables/) instead.
```

Comment:
Change:
To set up your credentials, run `mongocli config`, or use [env variables] (https://docs.mongodb.com/mongocli/stable/configure/environment-variables/) instead.